packages/libnvidia-container/0004-Use-NVIDIA_PATH-to-look-up-binaries.patch

```diff
@@ -0,0 +1,36 @@
+From 8cc6934b65f9b1c5c8b28844748c80520075c7c4 Mon Sep 17 00:00:00 2001
+From: Arnaldo Garcia Rincon <agarrcia@amazon.com>
+Date: Mon, 1 Nov 2021 16:18:19 +0000
+Subject: [PATCH 4/4] Use NVIDIA_PATH to look up binaries
+
+Signed-off-by: Arnaldo Garcia Rincon <agarrcia@amazon.com>
+---
+ src/nvc_info.c | 5 +++--
+ 1 file changed, 3 insertions(+), 2 deletions(-)
+
+diff --git a/src/nvc_info.c b/src/nvc_info.c
+index 3b3d2d5..8f954ff 100644
+--- a/src/nvc_info.c
++++ b/src/nvc_info.c
+@@ -249,8 +249,8 @@ find_binary_paths(struct error *err, struct dxcore_context* dxcore, struct nvc_d
+         char path[PATH_MAX];
+         int rv = -1;
+ 
+-        if ((env = secure_getenv("PATH")) == NULL) {
+-                error_setx(err, "environment variable PATH not found");
++        if ((env = secure_getenv("NVIDIA_PATH")) == NULL && (env = secure_getenv("PATH")) == NULL) {
++                error_setx(err, "environment variables NVIDIA_PATH/PATH not found");
+                 return (-1);
+         }
+         if ((env = ptr = xstrdup(err, env)) == NULL)
+@@ -281,6 +281,7 @@ find_binary_paths(struct error *err, struct dxcore_context* dxcore, struct nvc_d
+         }
+ 
+         while ((dir = strsep(&ptr, ":")) != NULL) {
++                log_infof("Looking at %s", dir);
```

Comment:
We probably don't actually want to have this log line here.  Once we have libnvidia-container working, this output is just noise.

Reply:
:sweat_smile: I saw this line after I did my last forced push. I already got rid off it.

Reply:
Is the change that removes this line coming at a later point? I thought I was looking at the current state of the patches.